DatadogRUM/Sources/RUM+Internal.swift

```diff
@@ -19,4 +55,56 @@ extension InternalExtension where ExtendedType == RUM {
     public static func isEnabled(in core: DatadogCoreProtocol = CoreRegistry.default) -> Bool {
         return core.get(feature: RUMFeature.self) != nil
     }
+
+    /// Enable URL session tracking after RUM has already been enabled. This method
+    /// is only needed if the configuration of URL session tracking is not known at initialization time,
+    /// or in the case of cross platform frameworks that do not initalize native URL session tracking.
+    ///
+    /// - Parameters:
+    ///    - in: the configuration for URL session tracking
+    public static func enableUrlSessionTracking(
+        with configuration: RUM.Configuration.LateURLSessionTracking,
+        in core: DatadogCoreProtocol = CoreRegistry.default) throws {
+        guard !(core is NOPDatadogCore) else {
+            throw ProgrammerError(
+                description: "Datadog SDK and RUM must be initialized before calling `RUM.enableUrlSessionTracking`."
+            )
+        }
+
+        guard let rum = core.get(feature: RUMFeature.self) else {
+            throw ProgrammerError(
+                description: "RUM must be initialized before calling `RUM.enableUrlSessionTracking`."
+            )
+        }
+
+        let distributedTracing: DistributedTracing?
+
+        // If first party hosts are configured, enable distributed tracing:
+        switch configuration.firstPartyHostsTracing {
+        case let .trace(hosts, sampleRate):
+            distributedTracing = DistributedTracing(
+                sampler: Sampler(samplingRate: sampleRate),
+                firstPartyHosts: FirstPartyHosts(hosts),
+                traceIDGenerator: configuration.traceIDGenerator
+            )
+        case let .traceWithHeaders(hostsWithHeaders, sampleRate):
+            distributedTracing = DistributedTracing(
+                sampler: Sampler(samplingRate: sampleRate),
+                firstPartyHosts: FirstPartyHosts(hostsWithHeaders),
+                traceIDGenerator: configuration.traceIDGenerator
+            )
+        case .none:
+            distributedTracing = nil
+        }
```

Comment:
/blocker The debug config should override the sample rate.
```suggestion
        // If first party hosts are configured, enable distributed tracing:
        switch configuration.firstPartyHostsTracing {
        case let .trace(hosts, sampleRate):
            distributedTracing = DistributedTracing(
                sampler: Sampler(samplingRate: configuration.debugSDK ? 100 : sampleRate),
                firstPartyHosts: FirstPartyHosts(hosts),
                traceIDGenerator: configuration.traceIDGenerator
            )
        case let .traceWithHeaders(hostsWithHeaders, sampleRate):
            distributedTracing = DistributedTracing(
                sampler: Sampler(samplingRate: configuration.debugSDK ? 100 : sampleRate),
                firstPartyHosts: FirstPartyHosts(hostsWithHeaders),
                traceIDGenerator: configuration.traceIDGenerator
            )
        case .none:
            distributedTracing = nil
        }
```